CHANGELOG.rst

```diff
@@ -0,0 +1,33 @@
+ifpress-solr-plugin CHANGELOG
+==================
+* 1,5,0
```

Comment:
I'm finishing up notes for the v1.4.0 now, will fix this comma issue with that commit